packages/engine/src/Actions.js

```diff
@@ -132,12 +131,24 @@ class Actions {
         params: parsedAction.params,
       });
     } catch (error) {
+      responses[action.id] = { error, index, type: action.type };
+      const { output: parsedMessages, errors: parserErrors } = this.context.parser.parse({
+        actions: responses,
+        event,
+        arrayIndices,
+        input: action.messages,
```

Comment:
Only really need to parse `action.messages.error`

Reply:
If we change this then test `operators are evaluated in params, skip and messages` fails. This means the `messages` field is not parsed entirely, but only a subset - the error field. As is, you could get the messages object with the schema fields.